### ISSUE TITLE
Fix the api

### ISSUE THREAD

**Issue by Caribosaurus** [thecsw/memeinvestor_bot#318]
The api is down.

**Comment by patrickblackjr** [comment]
This is a work in progress! Thanks for submitting.

**Comment by Caribosaurus** [comment]
pretty sure the issue is related to:
https://github.com/MemeInvestor/memeinvestor_bot/commit/af325324aabe30210c4cb5faa25007783f4682ab

if its not, it has to be related with architecture/hosting issue

**Comment by Caribosaurus** [comment]
also, the docker-compose has not been updated with the previously mentionned commit. Was the api down for 2 months?

**Comment by patrickblackjr** [comment]
I think we're going to be moving to a Node.js based API, since Node just works so fluidly in that manner.

**Comment by patrickblackjr** [comment]
See the repo here for the progress (really just boilerplate now): https://github.com/patrickblackjr/mib-api

**Comment by thecsw** [comment]
Hello there! Thank you for looking into the API issue. The binding that I changed from `0.0.0.0` to `192.168.0.1` is not the reason of the API's failure. The PEP8 compliant update was released and deployed before the firms update. API broke during first several days since firms update came out, meaning that the service was stable with new changes. I thought maybe some software got upgraded and became incompotible with other versions or deleted backwards compatibility, I blamed `gunicorn` but it turned out that none of our dependencies had breaking updates.
